doc.go

```diff
@@ -63,6 +63,25 @@ Or it can be loaded from a file using -f or --file:
 
 	$ yab -p localhost:9787 -t kv.thrift kv KeyValue::Get --file req.yaml
 
+Request options can also be specified in a YAML file, e.g., get.yab:
+	#!/usr/bin/env yab -y
```

Comment:
Yeah unfortunately there's no nice solution for Linux, filed https://github.com/yarpc/yab/issues/198

We have a couple of options, neither of which is great (using a separate `yaby` binary which assumes `-y`, or having `yab` detect with the first argument is a yaml template)